br/pkg/task/restore_ebs_meta.go

```diff
@@ -236,7 +236,22 @@ func (h *restoreEBSMetaHelper) restoreVolumes(progress glue.Progress) (map[strin
 			log.Error("failed to create all volumes, cleaning up created volume")
 			ec2Session.DeleteVolumes(volumeIDMap)
 		}
+
+		if h.cfg.UseFSR {
+			err = ec2Session.DisableDataFSR(h.metaInfo, h.cfg.TargetAZ)
+		}
 	}()
+
+	// Turn on FSR for TiKV data snapshots
+	if h.cfg.UseFSR {
+		err = ec2Session.EnableDataFSR(h.metaInfo, h.cfg.TargetAZ)
+		if err != nil {
+			return nil, 0, errors.Trace(err)
+		}
+
+		log.Info("FSR for data volume snapshots are enabled")
+	}
```

Comment:
We should wait all the fsr finished before restore volumes.

Reply:
fixed